utils/CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ include_directories (
     ${SNDFILE_INCLUDE_DIRS}
 )
 
+#adding installed getopt support for Windows 
+if ( WIN32 )
+    #let the user choosing additionnal search path
+    set (GETOPT_INCLUDE_PATH "d:\\freesw\\include\\getopt" 
```

Comment:
Sry, but hardcoding paths is bad practice. You should instead set `CMAKE_REQUIRED_INCLUDES` when calling cmake to configure the sources:

`cmake -DCMAKE_REQUIRED_INCLUDES="d:\\freews\\include\\getopt" ..`